applications/FluidDynamicsApplication/custom_processes/flows_measuring_process.cpp

```diff
@@ -0,0 +1,71 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Miguel Angel Celigueta
+//
+//
+
+// System includes
+#include <string>
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "processes/process.h"
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "containers/model.h"
+#include "includes/cfd_variables.h"
+#include "flows_measuring_process.h"
+#include "custom_utilities/post_process_utilities.h"
+
+
+namespace Kratos
+{
+
+
+    /// Constructor.
+    FlowsMeasuringProcess::FlowsMeasuringProcess(Model& rModel, Kratos::Parameters::Pointer pParameters) : mrModel(rModel)
+    {
+        Parameters default_parameters( R"(
+        {
+            "model_part_name" : "default_model_part_name",
+            "list_of_outlet_submodelpart_names" : []
```

Comment:
I would avoid the use of the `outlet` word as this can be equivalently used to measure inflow.